pkg/p2p/p2p.go

```diff
@@ -1,14 +1,14 @@
 // Copyright 2020 The Swarm Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
-
```

Comment:
This line should not be removed, as it is not the description of the package, but a copyright header.